Czech/RenX_Game.cze

```diff
@@ -0,0 +1,1598 @@
+//=====================================
+//	Game
+//=====================================
+[Rx_Game]
+Description="Znič základnu nepřítele, zatímco chráníš tu svoji."
+GameName="C&C Mode"
+EndOfMatchRulesTemplateStr_Scoring=První tým který zničí ten druhý vyhrává
+EndOfMatchRulesTemplateStr_Time=Tým s nejvíce body do `t minut vyhrává
+
+
+//=====================================
+//	Weapons 
+//=====================================
+
+[Rx_Weapon]
+ItemName="Bezejmenná Zbraň"
+PickupMessage="Bezejmenná Zbraň"
+
+[Rx_Weapon_Reloadable]
+ItemName="Bezejmenná Přebíjitelná Zbraň"
+PickupMessage="Bezejmenná Přebíjitelná Zbraň"
+
+[Rx_Weapon_Spray]
+ItemName="Bezejmenná Rozprašovací Zbraň"
+PickupMessage="Bezejmenná Rozprašovací Zbraň"
+
+[Rx_Weapon_Scoped]
+ItemName="Bezejmenná Zbraň s optikou"
+PickupMessage="Bezejmenná Zbraň s optikou"
+
+[Rx_Weapon_AutoRifle]
+ItemName="Automatická Puška"
+PickupMessage="Automatická Puška"
+
+[Rx_Weapon_TacticalRifle]
+ItemName="Taktická Puška"
+PickupMessage="Taktická Puška"
+
+[Rx_Weapon_Shotgun]
+ItemName="Brokovnice"
+PickupMessage="Brokovnice"
+			
+[Rx_Weapon_Shotgun_Crate]
+ItemName="Kinetická Brokovnice"	
+PickupMessage="Kinetická Brokovnice"
+
+[Rx_Weapon_RocketLauncher]
+ItemName="Raketomet"
+PickupMessage="Raketomet"
+
+[Rx_Weapon_Flamethrower]
+ItemName="Plamenomet"
+PickupMessage="Plamenomet"
+
+[Rx_Weapon_GrenadeLauncher]
+ItemName="Granátomet"
+PickupMessage="Granátomet"
+
+[Rx_Weapon_Pistol]
+ItemName="Tlumená Pistole"
+PickupMessage="Tlumená Pistole"
+
+[Rx_Weapon_Pistol_UnSILENCED]
+ItemName="Pistole"
+PickupMessage="Pistole"
+
+[Rx_Weapon_ChainGun]
+ItemName="Rotační Kulomet"
+PickupMessage="Rotační Kulomet"
+
+[Rx_Weapon_SniperRifle]
+ItemName="Odstřelovací Puška"
+PickupMessage="Odstřelovací Puška"
+
+[Rx_Weapon_VoltAutoRifle]
+ItemName="Voltová Automatická Puška"
+PickupMessage="Voltová Automatická Puška"
+
+[Rx_Weapon_RamjetRifle]
+ItemName="Ramjet Puška"
+PickupMessage="Ramjet Puška"
+
+[Rx_Weapon_Railgun]
+ItemName="Railgun"
```

Comment:
No Czech word for Railgun?

Reply:
No, the only translation possible is electromagnetic cannon in Czech but tha's too long.
And some words like skateboard for example don't have a Czech counterpart, English version is used instead.